### ISSUE TITLE
Input binds can not be set with on screen overlay buttons

### ISSUE THREAD

**Issue by FlukiestEmperor** [libretro/RetroArch#5494]
## Description

Input binds can not be set using the on screen game pad.

### Expected behavior

Binding an input should accept on screen keys.

### Actual behavior

I am only able to bind an input to a physical keyboard or physical controller.

### Steps to reproduce the bug

1. Enable onscreen overlay in menu
2. Go to Settings > Input > Input Hotkey binds or Input User Binds
3. Try to bind a key with an on screen button

### Version/Commit

- RetroArch: 1.6.7 / a235c0b

### Environment information

- OS: Tried this on Windows 10 and Android 7.1.1

**Comment by chrisisth** [comment]
is there really no way to assign any hotkey to a overlay after 9 years ?

**Comment by hizzlekizzle** [comment]
Yes, they use a different input system behind the scenes. You can assign hotkeys to overlays by editing the overlay's manifest/cfg file.

**Comment by chrisisth** [comment]
thanks for the answer i hope that gets worked on soon then. 
Or at least the rewind function gets its Retropad-Assignment option like the turbo function. 

Because having no autosave function because of the rewind function, which is inaccessible without editing cfg files leads to cutting out most mobile users to that, and many other functions.

**Comment by hizzlekizzle** [comment]
There are existing overlays that have dedicated rewind buttons on them, if that's what you're wanting: https://github.com/search?q=repo%3Alibretro%2Fcommon-overlays%20rewind&type=code
